src/config/plugin.ts

```diff
@@ -128,7 +131,7 @@ export class Plugin implements IPlugin {
 
     this.manifest = await this._manifest(Boolean(this.options.ignoreManifest), Boolean(this.options.errorOnManifestCreate))
     this.commands = Object.entries(this.manifest.commands)
-    .map(([id, c]) => ({...c, load: async () => this.findCommand(id, {must: true})}))
+    .map(([id, c]) => ({...c, pluginAlias: this.pluginAlias, pluginType: this.type, load: async () => this.findCommand(id, {must: true})}))
```

Comment:
Could not find a direct path to set pluginAlias or pluginType on a plugin instance, so applying those values when commands are extracted for the plugin.